src/lib/viz/style/helpers/color-categories-style.ts

```diff
@@ -19,6 +20,8 @@ export interface ColorCategoriesOptionsStyle extends Partial<BasicOptionsStyle>
   nullColor: string;
   // Color applied to features which the attribute value is not in the breaks.
   othersColor: string;
+  // Styling property.
+  property?: ColorsProperties;
```

Comment:
Nit. I would probably go for singular naming here for the type, so `ColorProperty` (and the same with `SizeProperty` type). What do you think?

Reply:
Done